buildspec.yml

```diff
@@ -34,6 +34,7 @@ phases:
       - sed -i "s,s3-region,$S3_REGION," includes/env.yml
       - sed -i "s,reply-by-email-address,replies+%{reply_key}@$D_HOSTNAME," includes/env.yml
       - sed -i "s,notification-email,notifications@$D_HOSTNAME," includes/env.yml
+      - sed -i "s,tests-passed,$CODE_REVISION," containers/app.yml
```

Comment:
I think currently this will just change the comment [here](https://github.com/mozilla/discourse.mozilla.org/blob/2a5159a/discourse-dev.yml#L11):

`## Which Git revision should this container use? (default: tests-passed)`

Because we're not doing a global replace. Perhaps we should be more explicit and do:

`s,version: tests-passed,$CODE_REVISION,`

Or we could change the line in the .yml files to be something like:

`version: code-revision # overwritten`

And then:

`s,code-revision,$CODE_REVISION,`

(this is the format I've been using in `env.yml`)

Reply:
that's a very good catch @LeoMcA, thank you!
I don't have strong opinions, but I prefer slightly more the second option you are proposing. Which one would you pick?

Reply:
I prefer the second option too!

Reply:
@LeoMcA I've updated the PR, please have a look